docs/using/admin.md

```diff
@@ -149,3 +149,17 @@ Urbit network.
 
 When this happens, back up any files you'd like to save, shut down your
 Urbit and recreate it (as if you were starting for the first time).
+
+## Creating SSL certificates
+
+It is possible to create SSL certificates so that you serve files through your ship's webserver using https.
+To do that you, would need a machine that has OpenSSL installed.
+In order to create a new private key and a self-signed certificate for your localhost, valid for a year, you can issue the following commands:
+
+```
+$ openssl req -x509 -sha256 -newkey rsa:4096 -keyout privatekey.pem -out certificate.pem -subj '/CN=localhost' -days 365
+$ mkdir -p your-urbit/.urb/tls
+$ mv privatekey.pem certificate.pem your-urbit/.urb/tls  
```

Comment:
The key should be `private.pem`. See https://github.com/urbit/urbit/blob/master/vere/http.c#L693